estilos.css

```diff
@@ -8,33 +8,53 @@
     display: flex;
     justify-content: space-between;
     width: 100%;
-    padding: 30px 50px;
-    background-color: rgb(190, 190, 190);
+    padding: 10px 10px;
+    background-color: black;
+    text-align: center;
 }
 
 .navbar__link {
-    color: grey;
-    font-size: 28px;
-    text-decoration: none;
+    width: 20%;
+    size: 38px;
```

Comment:
38px is not a valid value for 'size' property.
see documentation: [https://developer.mozilla.org/en-US/docs/Web/CSS/@page/size](url)